backend/src/news/schema.py

```diff
@@ -4,4 +4,8 @@ class NewsSumaryRequestSchema(BaseModel):
     content: str
 
 class PromptRequest(BaseModel):
-    prompt: str
\ No newline at end of file
+    prompt: str
+
+class NewsSumaryCustomModelSchema(BaseModel):
+    model: str
+    content: str
```

Comment:
Since `openai` and `anthropic` are the only 2 available options for the field `model`. You may consider use `Literal[...]` or `StrEnum` instead of simply `str` to help verify the received HTTP requests and simplify the conditional statements.

```suggestion
class NewsSumaryCustomModelSchema(BaseModel):
    model: Literal["openai", "anthropic"]
    content: str
```